pkg/api/grpcServer.go

```diff
@@ -87,33 +96,27 @@ func (s *grpcServer) dispatch(ctx context.Context, dState *dispatchState, worker
 		return makeGRPCError(out)
 	}
 
-	responseBag := attribute.GetMutableBag(nil)
+	preprocResponseBag := attribute.GetMutableBag(requestBag)
```

Comment:
does this get serialized anywhere? it'd be nice to dump this all in one place, perhaps close to where we log the response (for debug levels).